source/reference/operator/query/rand.txt

```diff
@@ -0,0 +1,63 @@
+=====
+$rand
+=====
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. query:: $rand
+
+:query:`$rand` has the following syntax:
```

Comment:
I would suggest reformatting slightly to  have the definition and then the syntax. That approach makes more sense to me as a reader.

```
:query:`$rand` returns a random float between 0 and 1. 

:query:`$rand` has the following syntax:

.. code-block:: javascript 

   { $rand: {} }
```

The formatting throughout our query operators is inconsistent, as you mentioned @davemungo. This is definitely something we should revisit as a whole and try to agree on a format to use throughout our operators.

Reply:
Done